src/main/flight/pid_init.c

```diff
@@ -292,8 +300,11 @@ void pidInitConfig(const pidProfile_t *pidProfile)
         pidRuntime.pidCoefficient[FD_YAW].Ki *= 2.5f;
     }
     pidRuntime.levelGain = pidProfile->pid[PID_LEVEL].P / 10.0f;
+    pidRuntime.levelFeedforwardGain = pidProfile->pid[PID_LEVEL].F / 200.0f;
```

Comment:
```suggestion
    // when faster, the feedforward gain needs to be a little higher
    pidRuntime.levelFeedforwardGain = pidProfile->pid[PID_LEVEL].F / 200.0f + pidProfile->level_response / 1000.0f;
    pidRuntime.level_response_gain = pidRuntime.dT * pidProfile->level_response / 10000.0f;
```

Reply:
The addition of extra angle feedforward when the time constant of the PT3 is faster is necessary because it is less effective at gaining angle otherwise.

Reply:
I need to look into this carefully. Having a filter parameter linked to a gain parameter and purposefully delaying a feedforward element all seems opposite to what control theory would suggest is the right approach. I'm also not sure this is  necessary with a correctly tuned D term which can directly address the wobble caused by FF. BF's acro controller works really well (and very much in keeping with theory), I'm keen to replicate that approach here and avoid going off book if at all possible. That said, if I can't get it to work right with a fast FF element then of course I'll eat a big portion of humble pie.